eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -54,6 +54,39 @@
       "RunAggregateReports": false,
       "TestGoals": "surefire:test",
       "TestOptions": ""
+    },
+    {
+      "Agent": {
+        "ubuntu-20.04": { "OSVmImage": "MMSUbuntu20.04", "Pool": "azsdk-pool-mms-ubuntu-2004-general" }
+      },
+      "JavaTestVersion": "1.17",
+      "AZURE_TEST_HTTP_CLIENTS": "jdk-httpclient",
+      "TestFromSource": true,
```

Comment:
I'm not sure if we need this to be a from source run. We don't have a from source run for OkHttp or Vertx.

Reply:
I see. We don't need it for JDK httpclient as well.